src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -186,10 +183,7 @@ const IfoCardTokens: React.FC<IfoCardTokensProps> = ({
         </Box>
       ) : null
 
-    if (
-      (ifo.version === 3 || (ifo.version === 3.1 && poolId === PoolIds.poolUnlimited)) &&
-      getBalanceNumber(credit) === 0
-    ) {
+    if (ifo.version === 3 || (ifo.version === 3.1 && poolId === PoolIds.poolUnlimited)) {
```

Comment:
can you add a comment here, we'll definitely need to add checking in the future

Reply:
comment added